engine/settings/celery_task_routes.py

```diff
@@ -156,9 +156,6 @@
     "apps.metrics_exporter.tasks.calculate_and_cache_user_was_notified_metric": {"queue": "long"},
     # SLACK
     "apps.integrations.tasks.notify_about_integration_ratelimit_in_slack": {"queue": "slack"},
-    "apps.slack.helpers.alert_group_representative.on_alert_group_action_triggered_async": {"queue": "slack"},
-    "apps.slack.helpers.alert_group_representative.on_alert_group_update_log_report_async": {"queue": "slack"},
-    "apps.slack.helpers.alert_group_representative.on_create_alert_slack_representative_async": {"queue": "slack"},
```

Comment:
removing these because `apps.slack.helpers` is not a valid directory and there's [no logs](https://ops.grafana-ops.net/goto/cVP4Q8jSR?orgId=1) over the past 30 days for `apps.slack.helpers`